tests/st/utils/utils.py

```diff
@@ -15,6 +15,7 @@
 import os
 import socket
 import sys
+from datetime import datetime
```

Comment:
the clean_calico_data should be updated to remove the UUID and creation timestamp fields - that way the assert_data doesn't need to worry about those values being set.